pkg/api/graph/test/hpa-missing-cpu-target.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: autoscaling/v1
+apiVersion: autoscaling/v2alpha1
```

Comment:
change doesn't match commit title. surprised this change was needed... shouldn't we prefer v1?